man/md2man-all.sh

```diff
@@ -18,7 +18,7 @@ for FILE in *.md; do
 	base="$(basename "$FILE")"
 	name="${base%.md}"
 	num="${name##*.}"
-	if [ -z "$num" -o "$name" = "$num" ]; then
+	if [[ -z "$num" || "$name" = "$num" ]]; then
```

Comment:
This seems like a weird use of `[[` IMO -- shouldn't this just be:

```suggestion
	if [ -z "$num" ] || [ "$name" = "$num" ]; then
```

Reply:
fixed